x/ccv/utils/utils.go

```diff
@@ -44,15 +44,6 @@ func AccumulateChanges(currentChanges, newChanges []abci.ValidatorUpdate) []abci
 	return out
 }
 
-func GetChangePubKeyAddress(change abci.ValidatorUpdate) (addr []byte) {
```

Comment:
Removed as it was called only from `ApplyCCValidatorChanges`. Added the logic directly there.